src/main/java/com/akamai/edgeauth/EdgeAuth.java

```diff
@@ -86,6 +86,24 @@ public class EdgeAuth {
     /** print all parameters. */
     private boolean verbose;
 
+    /**
+     * @param tokenType Not used
+     * @param tokenName Name for the new token
+     * @param key Secret required to generate the token. It must be hexadecimal digit string with even-length
+     * @param algorithm Algorithm to use to generate the token (sha1, sha256, or md5)
+     * @param salt Additional data validated by the token but NOT included in the token body. It will be deprecated
```

Comment:
Its written it will be deprecated, unless Im missing something I dont see it being used anywhere

Reply:
Similar case is for Node version https://github.com/akamai/EdgeAuth-Token-Node/blob/master/lib/edgeAuth.js , so I would not change it (yet)

Reply:
Do you think we should have a story to address that in both places?

Reply:
There are 5 similar projects, not all them are under Akamai space. We should address that in all projects at the same time once we move all of them into the space.